server/events/yaml/raw/global_cfg.go

```diff
@@ -18,11 +18,12 @@ type GlobalCfg struct {
 
 // Repo is the raw schema for repos in the server-side repo config.
 type Repo struct {
-	ID                   string   `yaml:"id" json:"id"`
-	ApplyRequirements    []string `yaml:"apply_requirements" json:"apply_requirements"`
-	Workflow             *string  `yaml:"workflow,omitempty" json:"workflow,omitempty"`
-	AllowedOverrides     []string `yaml:"allowed_overrides" json:"allowed_overrides"`
-	AllowCustomWorkflows *bool    `yaml:"allow_custom_workflows,omitempty" json:"allow_custom_workflows,omitempty"`
+	ID                   string         `yaml:"id" json:"id"`
+	ApplyRequirements    []string       `yaml:"apply_requirements" json:"apply_requirements"`
+	PreWorkflowHooks        []PreWorkflowHook `yaml:"pre_workflow_hooks" json:"pre_workflow_hooks"`
```

Comment:
I'm assuming we don't support this in the repo config currently?

Reply:
Not sure what would be the use case for repo config? For repo config there is a custom run commands.